src/styles/blocks/card.css

```diff
@@ -0,0 +1,106 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
Don't add empty lines at the start of the document.
```suggestion
@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```